src/um/pdh.rs

```diff
@@ -54,3 +208,338 @@ STRUCT!{struct PDH_COUNTER_PATH_ELEMENTS_W {
     szCounterName: LPWSTR,
 }}
 pub type PPDH_COUNTER_PATH_ELEMENTS_W = *mut PDH_COUNTER_PATH_ELEMENTS_W;
+
+EXTERN!{stdcall fn PdhGetDllVersion(
+    lpdwVersion: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhOpenQueryW(
+    szDataSource: LPCWSTR, dwUserData: DWORD_PTR, phQuery: *mut PDH_HQUERY
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhOpenQueryA(
+    szDataSource: LPCSTR, dwUserData: DWORD_PTR, phQuery: *mut PDH_HQUERY
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhAddCounterW(
+    hQuery: PDH_HQUERY, szFullCounterPath: LPCWSTR, dwUserData: DWORD_PTR,
+    phCounter: *mut PDH_HCOUNTER
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhAddCounterA(
+    hQuery: PDH_HQUERY, szFullCounterPath: LPCSTR, dwUserData: DWORD_PTR,
+    phCounter: *mut PDH_HCOUNTER
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhAddEnglishCounterW(
+    hQuery: PDH_HQUERY, szFullCounterPath: LPCWSTR, dwUserData: DWORD_PTR,
+    phCounter: *mut PDH_HCOUNTER
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhAddEnglishCounterA(
+    hQuery: PDH_HQUERY, szFullCounterPath: LPCSTR, dwUserData: DWORD_PTR,
+    phCounter: *mut PDH_HCOUNTER
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhCollectQueryDataWithTime(
+    hQuery: PDH_HQUERY, pllTimeStamp: *mut LONGLONG
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhValidatePathExW(
+    hDataSource: PDH_HLOG, szFullPathBuffer: LPCWSTR
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhValidatePathExA(
```

Comment:
doc is correct. Function calls that use TCHAR and TSTR varients expand to the *W or *A version of the function at the flip of the unicode switch in code. So it's not good practice to use TSTRs and TCHARs when using the *W or *A version specifically.